### PR TITLE
Store users

### PR DESCRIPTION
**Pull request by NDBman**

### CODE REVIEW

File: groupnews-dao/src/test/java/com/epam/internship/DaoIT.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+	@Test
+	@Transactional
+	public void count() {
+		assertEquals(2, systemUnderTest.count());
+	}
+
+	@Test
+	@Transactional
+	public void deleteOneOtherTwoShouldStay() {
```
Comment (uzonyib) [inline groupnews-dao/src/test/java/com/epam/internship/DaoIT.java]: Other one or two should stay?

File: groupnews-dao/src/test/java/com/epam/internship/DaoIT.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	public void deleteAllShouldLeaveNoUserEntites(){
+		//When
+		systemUnderTest.deleteAll();
+		//Then
+		assertEquals(0, systemUnderTest.count());
+	}
+	
+	@Test(expected=Exception.class)
+	@Transactional
+	public void setNameforNullShouldFail(){
```
Comment (uzonyib) [inline groupnews-dao/src/test/java/com/epam/internship/DaoIT.java]: Typo: setNameFor
I recommend finding a better name for the test which suggests which method of SUT is being tested, e.g. saveShouldFailForNullName.

File: groupnews-dao/src/test/java/com/epam/internship/DaoIT.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+	@Test
+	@Transactional
+	public void deleteAllShouldLeaveNoUserEntites(){
+		//When
+		systemUnderTest.deleteAll();
+		//Then
+		assertEquals(0, systemUnderTest.count());
+	}
+	
+	@Test(expected=Exception.class)
```
Comment (uzonyib) [inline groupnews-dao/src/test/java/com/epam/internship/DaoIT.java]: Missing spaces: expected = Exception.class

File: groupnews-service/src/main/java/com/epam/internship/config/GroupNewsServiceConfig.java [hunk 4]
```diff
@@ -1,10 +1,29 @@
...
+
+import com.epam.internship.converter.UserConverter;
+import com.epam.internship.converter.UserEntityConverter;
 
 @Configuration
-@ComponentScan(basePackages = { "com.epam.internship" })
 public class GroupNewsServiceConfig {
 
+	@Bean
+	public ConversionService conversionService(){
```
Comment (uzonyib) [inline groupnews-service/src/main/java/com/epam/internship/config/GroupNewsServiceConfig.java]: I think this is not needed: Spring will auto-create this bean and register all custom converters if you annotate your converter implementations with e.g. @Service.

File: groupnews-dao/src/test/java/com/epam/internship/DaoIT.java [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+import org.springframework.test.annotation.Rollback;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import com.epam.internship.entity.UserEntity;
+import com.epam.internship.repo.UserRepository;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootApplication
+@Rollback
+public class DaoIT {
```
Comment (Attila-Neuwirth) [inline groupnews-dao/src/test/java/com/epam/internship/DaoIT.java]: Name should reflect the class you are testing, like `UserRepositoryIT`.

File: groupnews-dao/src/test/java/com/epam/internship/DaoIT.java [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+		systemUnderTest.delete(1L);
+		//Then
+		assertEquals(1, systemUnderTest.count());
+		List<UserEntity> userEntites = systemUnderTest.findAll();
+		assertEquals(2L, userEntites.get(0).getId().longValue());
+	}
+
+	@Test
+	@Transactional
+	public void findOneShouldReturnUserEntityWithIdTest() {
```
Comment (Attila-Neuwirth) [inline groupnews-dao/src/test/java/com/epam/internship/DaoIT.java]: Please keep the nameing consistent, Test is not needed in method name.

File: groupnews-dao/src/test/java/com/epam/internship/DaoIT.java (hunk 5 above)
Comment (NDBman) [reply groupnews-dao/src/test/java/com/epam/internship/DaoIT.java]: Fixed it.

File: groupnews-dao/src/test/java/com/epam/internship/DaoIT.java (hunk 6 above)
Comment (NDBman) [reply groupnews-dao/src/test/java/com/epam/internship/DaoIT.java]: Fixed it

**Review by uzonyib — APPROVED** [review]

**Review by Attila-Neuwirth — APPROVED** [review]

**Pull request merged**